api/app/tests/hfi/test_hfi.py

```diff
@@ -72,19 +75,25 @@ def test_requested_fire_starts_unaltered(mocker: MockerFixture):
     """ Fire starts from user request remain unchanged """
 
     start_date = datetime.now()
-    daily = StationDaily(
-        code=1,
-        date=start_date,
-        intensity_group=1
-    )
-
     station = hfi_calc_models.PlanningWeatherStation(id=1, planning_area_id=1, station_code=1)
-
-    result = calculate_hfi_results(fire_start_ranges,
-                                   planning_area_fire_starts={
-                                       kamloops_fc.planning_areas[0].id: [fire_start_ranges[-1]]},
+    fuel_type_lookup = {
+        1: hfi_calc_models.FuelType(
+            id=1, abbrev='C1', description='C1', fuel_type_code='C1',
+            percentage_conifer=100, percentage_dead_fir=0)}
+    planning_area_fire_starts = {
+        kamloops_fc.planning_areas[0].id: [fire_start_ranges[-1]]}
+    wfwx_station = WFWXWeatherStation(
+        wfwx_id=1, code=1, name='station1', latitude=12.1,
+        longitude=12.1, elevation=123, zone_code=1)
+    raw_daily = {'stationId': 1, 'weatherTimestamp': datetime.now().timestamp() * 1000}
```

Comment:
`datetime.now()` gets mocked out for all tests correct?

Reply:
Yeah - this one doesn't require it - but I'll switch to get_utc_now anyway.